lib/src/builder.dart

```diff
@@ -75,9 +75,9 @@ class AggregateTestBuilder extends Builder {
   @override
   Future<void> build(BuildStep buildStep) async {
     _config ??= await decodeConfig(buildStep);
-    if (!_config.browserAggregation) {
+    if (!_config!.browserAggregation) {
```

Comment:
sort of annoying that _config can't be non-nullable, but checking the builder code it seems like that's just the way it has to be.

Reply:
I've improved this by using a local var in the latest commit.